clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -1401,41 +1401,47 @@ def VTableAddrPointOp : CIR_Op<"vtable.address_point",
 }
 
 //===----------------------------------------------------------------------===//
-// StructElementAddr
+// GetMemberOp
 //===----------------------------------------------------------------------===//
 
-// FIXME: rename this among the lines of GetGlobalOp.
-def StructElementAddr : CIR_Op<"struct_element_addr"> {
+def GetMemberOp : CIR_Op<"get_member"> {
   let summary = "Get the address of a member of a struct";
   let description = [{
-    The `cir.struct_element_addr` operaration gets the address of a particular
-    named member from the input struct.
+    The `cir.get_member` operation gets the address of a particular named
+    member from the input record.
 
-    It expects a pointer to the base struct as well as the name of the member
+    It expects a pointer to the base record as well as the name of the member
     and its field index.
 
     Example:
     ```mlir
-    !ty_22struct2EBar22 = type !cir.struct<"struct.Bar", i32, i8>
-    ...
-    %0 = cir.alloca !ty_22struct2EBar22, cir.ptr <!ty_22struct2EBar22>
-    ...
-    %1 = cir.struct_element_addr %0, "Bar.a"
-    %2 = cir.load %1 : cir.ptr <int>, int
-    ...
+    // Suppose we have a struct with multiple members.
+    !s32i = !cir.int<s, 32>
+    !s8i = !cir.int<s, 32>
+    !struct_ty = !cir.struct<"struct.Bar" {!s32i, !s8i}>
+
+    // Get the address of the member at index 1.
+    %1 = cir.get_member %0[1] {name = "i"} : (!cir.ptr<!struct_ty>) -> !cir.ptr<!s8i>
     ```
   }];
 
   let arguments = (ins
-    Arg<CIR_PointerType, "the address to load from", [MemRead]>:$struct_addr,
-    StrAttr:$member_name,
-    IndexAttr:$member_index);
+    Arg<CIR_PointerType, "the address to load from", [MemRead]>:$addr,
+    StrAttr:$name,
+    IndexAttr:$indexAttr);
```

Comment:
I choose to leave it as `indexAttr` because it would conflict with the existing `getIndex` getter that fetches the index value as an unsigned integer.

Reply:
Just noticed that this name should use snake case (e.g. `index_attr`) to conform with MLIR generic format.